### DISCUSSION TITLE
PTZ cameras in forthcoming v0.13?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#7458] [General]
Will the forthcoming Frigate v0.13 work with the Dahua PTZ1A225-HNR-XA? This camera is incredible and I would like to get one if it will work with Frigate.

**Comment by NickM-27** [comment]
Does it support ptz via onvif?

**Reply by Cold-Lemonade** [reply]
My understanding is that there isn't a strict ONVIF protocol for PTZ functionality. This particular camera is compatible with DH-SD and Pelco-P/D.

**Reply by NickM-27** [reply]
RIght, but it has to support ptz via onvif in general, which is seems like it does
